app/controllers/v0/institutions_controller.rb

```diff
@@ -60,7 +60,6 @@ def search_results
       @query ||= normalized_query_params
```

Comment:
@mphprogrammer Do the normalized query params list caution_flag as an available query param? I think we should also maybe remove it from that array constant, if applicable.